client/app/house/components/RemoveMemberAlertModal.tsx

```diff
@@ -0,0 +1,73 @@
+import {
+  AlertDialog,
+  AlertDialogAction,
+  AlertDialogCancel,
+  AlertDialogContent,
+  AlertDialogDescription,
+  AlertDialogFooter,
+  AlertDialogHeader,
+  AlertDialogTitle,
+} from '@/_components/ui'
+import { HouseType } from '@/_types'
+import { useRemoveMember } from '../hooks'
+
+type RemoveMemberAlertModalProps = {
+  house: HouseType
+  isOpenAlert: boolean
+  setIsOpenAlert: (isOpenAlert: boolean) => void
+  setIsOpenModal: (isOpenModal: boolean) => void
+  removedMemberId: number | null
+}
+
+const RemoveMemberAlertModal = ({
+  house,
+  isOpenAlert,
+  setIsOpenAlert,
+  setIsOpenModal,
+  removedMemberId,
+}: RemoveMemberAlertModalProps) => {
+  const { onRemove, isPending: isRemoving } = useRemoveMember()
+  const houseId = house.houseId
+
+  const handleRemoveMember = () => {
+    if (!removedMemberId) return
+    onRemove(removedMemberId, houseId)
+
+    setIsOpenAlert(false)
+    setIsOpenModal(true)
+  }
+
+  const handleCancel = () => {
+    setIsOpenModal(false)
+    setIsOpenModal(true)
+  }
+
+  return (
+    <AlertDialog open={isOpenAlert} onOpenChange={() => setIsOpenAlert(false)}>
+      <AlertDialogContent>
+        <AlertDialogHeader>
+          <AlertDialogTitle>
+            Are you certain you wish to remove this member?
+          </AlertDialogTitle>
```

Comment:
I think it's better to write with just one line, like house delete! We should show the same way for deleting modal. 
If you do like so, we can show `Are you absolutely sure?` as the title, and details in the explaining area. What do you think? 

<img width="559" alt="Screenshot 2024-04-02 at 4 40 05 PM" src="https://github.com/HAK2024/fairy-share/assets/66394413/2bff85fc-30dc-40a4-b1fd-c05804de7f7b">

Reply:
Fixed!
https://github.com/HAK2024/fairy-share/pull/93/commits/5f4d7b326b730d2eedea1c202aa09542975be2e9